src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -169,17 +183,32 @@ class WorkspacePeoplePage extends React.Component {
     renderItem({
         item,
     }) {
+        const initialDimensions = Dimensions.get('window');
+        const isSmallScreenWidth = initialDimensions.width <= variables.mobileResponsiveWidthBreakpoint;
+        const isMobileDevice = getPlatform() === CONST.PLATFORM.ANDROID || getPlatform() === CONST.PLATFORM.IOS || isSmallScreenWidth;
```

Comment:
Reminder for us to test this on iPad, as I'm not sure whether `isSmallScreenWidth` will include large iPads.